blocks/commerce-cart/commerce-cart.js

```diff
@@ -79,7 +87,15 @@ export default async function decorate(block) {
     // Cart List
     provider.render(CartSummaryList, {
       hideHeading: hideHeading === 'true',
-      routeProduct: (product) => rootLink(`/products/${product.url.urlKey}/${product.topLevelSku}`),
+      routeProduct: (product) => {
+        const path = `/products/${product.url.urlKey}/${product.topLevelSku}`;
+        const url = new URL(rootLink(path), window.location.origin);
+        url.searchParams.set('itemUid', product.uid);
+        if (window.Cypress) {
+          return path;
+        }
+        return url.toString();
+      },
```

Comment:
As we discussed, since this would like to use itemUid, PR #419 has been updated to provide this in the redirected cart page. So we could revert this part to the original and rely on PR#419 to obtain the itemUid to display the message. 
Sample redirected cart page URL after user updates a configurable product: https://usf-2008--aem-boilerplate-commerce--hlxsites.aem.live/cart?itemUid=NjExMQ%3D%3D

```suggestion
routeProduct: (product) => rootLink(`/products/${product.url.urlKey}/${product.topLevelSku}`),
```